blocks/editable/index.js

```diff
@@ -207,6 +207,8 @@ export default class Editable extends wp.element.Component {
 			event.stopImmediatePropagation();
 		}
 
+		// If we click shift+Enter on inline Editables, we should avoid creating `p` tags
```

Comment:
It would also split two contenteditables into two, e.g. a heading will suddenly become two headings.